synapse/rest/client/v1/admin.py

```diff
@@ -616,6 +616,69 @@ def on_GET(self, request, target_user_id):
         defer.returnValue((200, ret))
 
 
+class ServerNoticeRestServlet(ClientV1RestServlet):
```

Comment:
it seems like it is actually `v1`

Reply:
It is that class but it adds the `/client/r0/` endpoints as well.